templates/vue/src/components/Lightbox/index.vue

```diff
@@ -16,6 +16,7 @@
       :node="node"
       :row-id="rowId"
       :sub-row-id="subRowId"
+      :activeMenuNode="activeMenuNode"
```

Comment:
Rather than doing this, couldn't you simply change the `node` prop here to be either the original node or the active menu node? (maybe create a computed property that uses the activeMenuNodeId to return which node it should display) That way, you theoretically wouldn't have to change the multiContentMedia  component.